README.md

```diff
@@ -68,6 +68,8 @@ Bookstore is app that I am going to create using react and redux. I will update
 
```

Comment:
![image](https://github.com/Atifzada/BookStore_App/assets/93039359/0356119a-c0db-401d-b397-1e0bf3fdd93b)

**[OPTIONAL]**

- [ ] It would be nice if you add some meaningful features in the Future Feature section to make your readme descriptive and professional 😊

Reply:
Thank you for the helpful review, surely I will come up with solutions